robot/impl/local_robot_test.go

```diff
@@ -3420,3 +3281,16 @@ func TestCloudMetadata(t *testing.T) {
 		})
 	})
 }
+
+func initTestRobot(t *testing.T, cfg *config.Config) (robot.LocalRobot, *observer.ObservedLogs, func()) {
```

Comment:
following [this proposed pattern for test helpers](https://sourcegraph.com/blog/go/advanced-testing-in-go#test-helpers)

> Never return errors. Just pass in *testing.T and fail.